webapp/src/Controller/API/ContestController.php

```diff
@@ -417,13 +417,25 @@ public function getContestYamlAction(Request $request, string $cid): StreamedRes
         $response->setCallback(function () use ($contest, $penalty_time) {
             echo "name:                     " . $contest->getName() . "\n";
             echo "short-name:               " . $contest->getExternalid() . "\n";
+            echo "activation-time:               " .
+                Utils::absTime($contest->getActivatetime(), true) . "\n";
             echo "start-time:               " .
                 Utils::absTime($contest->getStarttime(), true) . "\n";
             echo "duration:                 " .
                 Utils::relTime($contest->getEndtime() - $contest->getStarttime(), true) . "\n";
             echo "scoreboard-freeze-length: " .
                 Utils::relTime($contest->getEndtime() - $contest->getFreezetime(), true) . "\n";
             echo "penalty-time:             " . $penalty_time . "\n";
+            echo "freeze-time:               " .
+                Utils::absTime($contest->getFreezetime(), true) . "\n";
+            echo "end-time:               " .
+                Utils::absTime($contest->getEndtime(), true) . "\n";
+            echo "finalize-time:               " .
+                Utils::absTime($contest->getFinalizetime(), true) . "\n";
+            echo "unfreeze-time:               " .
```

Comment:
We may want to call this `scoreboard_thaw_time` to already prepare for https://github.com/icpc/ccs-specs/pull/134/files#diff-353f5220dc395fec383c89d1431e3c5733913a1af93f9e8d3c0135d49b2fb5ebR735